.buildkite/packaging.pipeline.yml

```diff
@@ -265,6 +334,76 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "STAGING: {{matrix}} FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
+          SNAPSHOT: false
+          DEV: false
+          FIPS: true
+        command: |
+          source .buildkite/scripts/version_qualifier.sh
+          .buildkite/scripts/packaging/package-dra.sh {{matrix}}
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+
+      - label: "STAGING: {{matrix}} Linux/arm64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_ARM_FIPS}"
+          SNAPSHOT: false
+          DEV: false
```

Comment:
Missed the comment about using `DEV: false`, wonder if this is needed for all the jobs, what's the reason is not added as a top-level env variable?

Reply:
I took the exact same env var settings from the already existing packaging steps (including the comment).
As a general runle, we don't want to generate DRAs with DEV=true because DEV builds usually are for local development/debugging and not for something that can be released.

As for the reason why it's not a top level env var, I cannot say for sure but I am guessing this is some leftover from the Jenkins->BK migration.
@pazone @dliappis may know more about the specific circumstances

Reply:
`DEV` is always `false` except [one step](https://github.com/elastic/beats/blob/df3d1077ccb3df096aa2c4d8c86cdac78bd517e6/.buildkite/packaging.pipeline.yml#L84). Hence, we can't declare it as `false` on pipeline level. Unfortunately `group` doesn't have `env:` field.